packages/snaps-utils/src/constants.ts

```diff
@@ -0,0 +1,4 @@
+// The max file size for Snap source code and auxiliary files.
+// 64 MB - we chose this number because it is the size limit for postMessage
+// between the extension and the offscreen document enforced by Chrome.
+export const MAX_FILE_SIZE = 64_000_000;
```

Comment:
Should we have a small difference here to account for the JSON message itself?

Reply:
The actual limit in Chrome is slightly larger, so I think we already have a decent buffer: https://source.chromium.org/chromium/chromium/src/+/main:extensions/renderer/api/messaging/messaging_util.cc;l=68;drc=197cb5f6af77ae63d1676eeb61a806a7fef06e6d